rerun_py/src/utils.rs

```diff
@@ -23,3 +23,26 @@ where
     let runtime: &Runtime = get_tokio_runtime();
     py.allow_threads(|| runtime.block_on(f))
 }
+
+pub fn py_rerun_warn(msg: &std::ffi::CStr) -> PyResult<()> {
+    Python::with_gil(|py| {
+        let warning_type = PyModule::import(py, "rerun")?
+            .getattr("error_utils")?
+            .getattr("RerunWarning")?;
+        PyErr::warn(py, &warning_type, msg, 0)?;
+        Ok(())
+    })
+}
+
+pub fn py_log_warn(msg: &str) -> PyResult<()> {
+    re_log::warn!(msg);
+    let csmsg;
+    let cmsg = match CString::new(msg) {
+        Ok(csm_temp) => {
+            csmsg = csm_temp;
+            csmsg.as_c_str()
+        }
+        _ => c"Warning: :Failed to get warning message.",
```

Comment:
if we'd ever hit this this is very unlikely to be caught. I think it would be better to just map the error to `PyResult` and return

Reply:
Noted